bptree_test.go

```diff
@@ -544,3 +544,37 @@ func TestBPTree_WriteNode(t *testing.T) {
 	// remove the test file
 	_ = os.Remove(testFilename)
 }
+
+func TestEnqueueDequeue(t *testing.T) {
+	// 初始化测试数据
+	node1 := &Node{KeysNum: 1}
+	node2 := &Node{KeysNum: 2}
+	node3 := &Node{KeysNum: 3}
+
+	// 测试 enqueue() 函数
+	enqueue(node1)
+	enqueue(node2)
+	enqueue(node3)
+
+	// 测试 dequeue() 函数，并验证结果
```

Comment:
Could you help translate the comments into English? Thanks!

Reply:
ok